sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2018,28 +2021,32 @@ class DataSourceV2SQLSuite
     }
   }
 
-  test("CACHE TABLE") {
+  test("CACHE/UNCACHE TABLE") {
```

Comment:
shall we also test CACHE TABLE AS SELECT? with temp view exists and not.

Reply:
Added a test with an existing temp view to `CachedTableSuite`.